<launch>
    <arg name="bag_path" default="/home/dji/swarm_log_lastest" />
    <node pkg="swarm_vo_fuse" name="swarm_vo_fuse" type="swarm_vo_fuse_node" output="screen">
       <param name="force_freq" value="10" type="double" />
       <param name="max_accept_cost" value="0.15" type="double" />
       <param name="max_keyframe_num" value="100" type="int" />
       <param name="min_keyframe_num" value="10" type="int" />
       <param name="thread_num" value="4" type="int" />
    </node>
</launch>
